benchmarks.py

```diff
@@ -39,3 +40,13 @@ def bootstrap_agent(task):
         triggering_prompt=DEFAULT_TRIGGERING_PROMPT,
         workspace_directory=str(workspace_directory_path),
     )
+
+
+def get_command_registry(config):
+    command_registry = CommandRegistry()
+    enabled_command_categories = [
+        x for x in COMMAND_CATEGORIES if x not in config.disabled_command_categories
```

Comment:
If someone changes this in their .env it will change the benchmarks and tests and such. Should we ignore the disabled categories for tests?

Reply:
if you remove it, it means that: 
- the benchmark will run with the abilities we remove by default
- but users have to actually activate this ability in order to get it.

This means the default mode is with the default disabled_command_categories activated.

The CI pipeline runs with the right .env anyway so we're good.

But you know like me that we need a better to overwrite this .env